### PR TITLE
Make variable name consistent

### PR DESCRIPTION
**Pull request by markapruett**
*Issue #, if available:*

*Description of changes:*
The variable is called `ARTIFACTS_BUCKET` and `ARTIFACT_BUCKET` in different places. Add a check to ensure it's the same when used

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Common.mk [hunk 1]
```diff
@@ -26,7 +26,7 @@ OUTPUT_BIN_DIR?=$(OUTPUT_DIR)/bin/$(REPO)
 #################### AWS ###########################
 AWS_REGION?=us-west-2
 AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
-ARTIFACT_BUCKET?=my-s3-bucket # TODO: in eks-a we use ARTIFACTS_BUCKET
+ARTIFACTS_BUCKET?=my-s3-bucket
```
Comment (danbudris) [inline Common.mk:29]: is there any place, such as in the Prow jobs, where we're over-riding this variable using the `ARTIFACT_BUCKET` name?

Comment (markapruett) [reply Common.mk:29]: Yes -- loads. Good catch! I'll update there

Comment (markapruett) [reply Common.mk]: Updated to avoid changing in so many places at once

**Pull request merged**